File: 3263-divide-an-array-into-subarrays-with-minimum-cost-i/divide-an-array-into-subarrays-with-minimum-cost-i.java
class Solution {
    public int minimumCost(int[] nums) {
        int n = nums.length;
        int mn = Integer.MAX_VALUE;
        int first = nums[0];
        
        for (int i = 1; i < n - 1; i++) {
            for (int j = i + 1; j < n; j++) {
                mn = Math.min(nums[i] + nums[j], mn);
            }
        }
        
        return first + mn;
    }
}

File: 3263-divide-an-array-into-subarrays-with-minimum-cost-i/divide-an-array-into-subarrays-with-minimum-cost-i.cpp
class Solution {
public:
    int minimumCost(vector<int>& nums) {
        int n = nums.size();
        int mn = INT_MAX ;
        int first = nums[0];
        for(int i = 1 ;i < n-1; i++ ){
            for(int j= i+1; j< n;j++){
            mn= min(nums[i]+nums[j],mn);
            }
        }
        return first+mn;
    }
};